### ISSUE TITLE
CU Syndicated Content Block - DOM format

### ISSUE THREAD

**Issue by jcsparks** [CuBoulder/tiamat-theme#1696]
We want to make a couple of changes to the DOM formatting of the CU Syndicated Content block.  

First, the content appears correct visually but having the image display first is sub-optimal for screen readers.  Change the order of the content in the DOM to show Label then Body then Image and use flexbox to reverse the order as necessary to keep the content displaying viusal as is.  This should be the default for each format (teaser and feature as well as any future formats) for a11y reasons.  

Additionally for a11y reason, we don't want the Label to be using a header tag.  Switch this out to a Label tag or other appropriate HTML element and add some CSS to style like the H3 tag that is being used currently.  Nothing else should change about the visual display or link functionality, just need to switch from using an H3 tag.
